.github/workflows/test.yml

```diff
@@ -6,8 +6,17 @@ jobs:
   test-react:
     name: Test React
     runs-on: ubuntu-latest
+    container:
+      image: cypress/browsers:node14.17.0-chrome91-ff89
+      options: --privileged
 
     steps:
+      - name: Upgrade git
+        run: |
+          echo deb "http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list
+          apt-get update
+          apt-get install -y git/buster-backports
```

Comment:
by default the cypress container only uses git v2.20, but the `dorny/paths-filter@v2` action requires 2.22, so this upgrades git.